device/core/src/iotedged_authentication_provider.ts

```diff
@@ -0,0 +1,131 @@
+import { AuthenticationProvider, TransportConfig, encodeUriComponentStrict } from 'azure-iot-common';
+import { SharedAccessKeyAuthenticationProvider } from './sak_authentication_provider';
+import { RestApiClient } from 'azure-iot-http-base';
+
+// tslint:disable-next-line:no-var-requires
+const packageJson = require('../package.json');
+
+export const WORKLOAD_API_VERSION = '2018-06-28';
+const DEFAULT_SIGN_ALGORITHM = 'HMACSHA256';
+const DEFAULT_KEY_ID = 'primary';
+
+export interface EdgedAuthConfig {
```

Comment:
>export [](start = 0, length = 6)

for all exported members there needs to be a /** @private */ jsdoc comment to avoid surfacing them in the auto-generated docs on docs.microsoft.com

Reply:
Sure. Will add that.